# (PART) FIELD PROCEDURES {-}
# About fieldwork {#about-fieldwork}

:::: {.graybox}
Some contents of this section have been extracted and modified from the article [Field and laboratory guidelines for reliable bioinformatic and statistical analysis of bacterial shotgun metagenomic data](https://www.nature.com/articles/s41576-021-00421-0) published in ***Critical Reviews in Biotechnology*** in 2023 by the authors of the **Holo-omics Workbook**.
:::

# Sample collection {#fieldwork-sample-collection}
Sample collection is a nuanced process that necessitates careful consideration before embarking into the field. Determining what samples to gather, when to obtain them, and how to procure them are three critical decisions that should be made in advance to ensure an effective sampling endeavour.

Researchers must bear in mind that due to the intricate and dynamic nature of microbial communities, any samples acquired may already deviate from their original state [@Probandt2018-mn]. This discrepancy can be attributed to temporal shifts, spatial disparities, and distinctions between the sampled substrate and the subject of study. An illustrative instance is the collection of faecal samples, which is notorious for not perfectly representing the true community within the lower intestine [@Yan2019-nb].

Furthermore, if faecal samples are sourced from the environment rather than directly from the host animal, the microbial communities within them may undergo alterations due to shifts in physicochemical parameters (such as oxygen levels and temperature) caused by exposure to air and sunlight [@Fofanov2018-wu]. Additionally, colonisation by new bacteria from the surrounding environment can influence the samples. Hence, it is advisable to obtain faecal samples directly from the animals or from sterile containers in which animals are temporarily housed, thereby minimising environmental influences.

It is important to acknowledge that samples typically exhibit spatial structure [@Ji2019-wj]. Consequently, when subsampling is performed, the composition of the subset will depend on the precise location (down to the micron level) from which the sample was taken. In cases where acquiring faecal samples from the environment is unavoidable, biases introduced by exposure can be mitigated by targeting internal subsamples and avoiding external layers. However, this approach might introduce its own spatial biases [@Griffin2021-pt].

Dealing with sizeable samples often imposes constraints related to DNA preservation and extraction methods, limiting the amount of material that can be processed. As a result, researchers may consider gathering multiple small subsamples using either random or structured subsampling strategies. This approach captures the spatial diversity of the sample, as opposed to relying on a single large sample. Subsequently, these subsamples could be aggregated to represent the entire sample or, ideally, processed individually as distinct biological replicates. This practice could potentially enhance metagenomic binning procedures.


:::: {.authorbox}
Contents of this section were created by [Antton Alberdi](#antton-alberdi) and [Ostaizka Aizpurua](#ostaizka-aizpurua).  
:::

# Sample preservation {#fieldwork-sample-preservation}
Utilising liquid nitrogen for snap-freezing samples is widely regarded as the benchmark method for extracting pristine DNA from an unmodified microbial community. Nevertheless, the feasibility of promptly freezing samples while maintaining an unbroken cold chain isn't always achievable, especially during field sampling missions. Consequently, dependable substitutes become imperative. This necessity has spurred the creation of various preservatives, enabling samples to be preserved at room temperature over prolonged durations. Nonetheless, research has unveiled that these preservatives can impart significant biases during the generation of diverse omic layers [@Bjerre2019-mb, @Perez-Losada2016-nc]. Therefore, maintaining uniformity in the selection of preservatives is of paramount importance to ensure consistency across analyses.

|Preservative|HG/MG|HT/MT|MP|ME|Reference|
|---|---|---|---|---|---|
| Snap frozen | Yes | Yes | Yes | Yes | [@De_Spiegeleer2020-qc] |
| RNAlater | Yes | Yes | Yes | No | [@Van_Eijsden2013-dx] |
| DNA/RNA Shield | Yes | Yes | No | No | [@Schweighardt2015-xk] |
| OMNIgene GUT | Yes | Yes | No | No | [@Wang2018-fs] |
| Tris-EDTA Buffer | Yes | Yes | No | No | [@Barra2015-nv] |
| Guanidine thiocyanate | Yes | Yes | No | No | [@Weidner2022-da] |
| TRIzol | Yes | Yes | Yes | No | [@Simoes2013-xh] |
| Protease inhibitors | No | No | Yes | No | [@Ryan2017-pl] |
| FTA Cards | Yes | Yes | No | Yes | [@Bolt_Botnen2023-ql] |
| Methanol | No | No | No | Yes | [@Straughen2023-ia] |
| Ammonium bicarbonate | No | No | Yes | No | @Hedges2013-ip |

Numerous preservatives necessitate removal before extraction, such as ethanol, NAP buffer, and RNAlater. However, this removal process can inadvertently eliminate non-pelleting entities, including bacteriophages and other viruses. Conversely, certain preservatives also serve as lysis buffers, exemplified by Zymoâ€™s DNA/RNA Shield. Notably advantageous, these buffers directly participate in DNA extraction. Although these preservatives do not stabilise DNA within cells, they initiate cellular degradation while stabilising DNA in the matrix. Maintaining the recommended material-to-buffer ratio is imperative across all these buffers to ensure optimal outcomes. Overloading with biological material can negate the beneficial effects of the buffers.

Avoiding freeze-thaw cycles is ideal, as they are recognised sources of DNA degradation and variations in microbial community composition, especially when samples are repetitively thawed [@Cuthbertson2015-uy]. Opting for small aliquots tailored to the extraction protocol during sample collection, rather than bulk collection, facilitates thawing only the sample intended for processing. This practice sidesteps detrimental thaw-freeze cycles and diminishes the risk of cross-contamination from other samples.

Furthermore, the biological and chemical characteristics of molecules (e.g., DNA, RNA, proteins, metabolites) used in omic data generation must be acknowledged. Host DNA's abundance and stability render HG less sensitive. Conversely, MG warrants more cautious handling due to potential microbial community fluctuations post-sampling, unless biochemical reactions are halted. HT and MT demand even swifter preservation to capture representative gene expression patterns. Lastly, metabolites exhibit diverse chemical properties, ranging from stable steroids to highly volatile short-chain fatty acids. Thus, judicious selection of appropriate preservatives becomes paramount when generating multiple omic layers. This decision involves determining whether omic data will be sourced from a single biological sample, necessitating a universal preservative, or multiple samples, each potentially requiring a distinct preservative.

Importantly, the diverse physicochemical properties of samples mandate that collection and storage methods validated for one sample type cannot be universally assumed optimal for others. Therefore, preliminary optimisation tests are prudent, and methodological consistency emerges as a prerequisite for the production of dependable omic data.

:::: {.authorbox}
Contents of this section were created by [Antton Alberdi](#antton-alberdi),  [Ostaizka Aizpurua](#ostaizka-aizpurua) and [Jacob A Rasmussen](#jacob-rasmussen).  
:::
