<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>5 Sample preservation | A Practical Guide to Holo-Omics</title>
  <meta name="description" content="5 Sample preservation | A Practical Guide to Holo-Omics" />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="5 Sample preservation | A Practical Guide to Holo-Omics" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="5 Sample preservation | A Practical Guide to Holo-Omics" />
  
  
  

<meta name="author" content="The Center for Evolutionary Hologenomics" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="fieldwork-sample-collection.html"/>
<link rel="next" href="about-labwork.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-45DK1TCMQC"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-45DK1TCMQC');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this guidebook</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contents"><i class="fa fa-check"></i>Contents</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#protocols-exercises-tutorials"><i class="fa fa-check"></i>Protocols, exercises and tutorials</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#datasets"><i class="fa fa-check"></i>Data sets</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#about-the-authors"><i class="fa fa-check"></i>About the authors</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-cite-this-work"><i class="fa fa-check"></i>How to cite this work</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgement"><i class="fa fa-check"></i>Acknowledgement</a></li>
</ul></li>
<li class="part"><span><b>I INTRODUCTION</b></span></li>
<li class="chapter" data-level="1" data-path="holo-omics.html"><a href="holo-omics.html"><i class="fa fa-check"></i><b>1</b> Introduction to holo-omics</a>
<ul>
<li class="chapter" data-level="" data-path="holo-omics.html"><a href="holo-omics.html#why-holo-omics"><i class="fa fa-check"></i>Why do we need holo-omics?</a></li>
<li class="chapter" data-level="" data-path="holo-omics.html"><a href="holo-omics.html#what-is-holo-omics"><i class="fa fa-check"></i>What is holo-omics?</a></li>
<li class="chapter" data-level="1.1" data-path="omic-layers.html"><a href="omic-layers.html"><i class="fa fa-check"></i><b>1.1</b> Omic layers</a>
<ul>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-genomics"><i class="fa fa-check"></i>Host genomics (HG)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-transcriptomics"><i class="fa fa-check"></i>Host transcriptomics (HT)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metagenomics"><i class="fa fa-check"></i>Microbial metagenomics (MG)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metatranscriptomics"><i class="fa fa-check"></i>Microbial metatranscriptomics (MT)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-proteomics"><i class="fa fa-check"></i>Host proteomics (HP)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metaproteomics"><i class="fa fa-check"></i>Microbial metaproteomics (MP)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#meta-metabolomics"><i class="fa fa-check"></i>(Meta)metabolomics (ME)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="study-design-considerations.html"><a href="study-design-considerations.html"><i class="fa fa-check"></i><b>2</b> Study design considerations</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hologenomic-complexity.html"><a href="hologenomic-complexity.html"><i class="fa fa-check"></i><b>2.1</b> Hologenomic complexity</a></li>
<li class="chapter" data-level="2.2" data-path="control-of-variables.html"><a href="control-of-variables.html"><i class="fa fa-check"></i><b>2.2</b> Control of variables</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-host-genomes"><i class="fa fa-check"></i><b>2.2.1</b> Controlling host genomes</a></li>
<li class="chapter" data-level="2.2.2" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-microbial-metagenomes"><i class="fa fa-check"></i><b>2.2.2</b> Controlling microbial metagenomes</a></li>
<li class="chapter" data-level="2.2.3" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-environment"><i class="fa fa-check"></i><b>2.2.3</b> Controlling the environment</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="molecular-resolution.html"><a href="molecular-resolution.html"><i class="fa fa-check"></i><b>2.3</b> Molecular resolution</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="molecular-resolution.html"><a href="molecular-resolution.html#host-genotype-resolution"><i class="fa fa-check"></i><b>2.3.1</b> Resolution of host genotypes</a></li>
<li class="chapter" data-level="" data-path="molecular-resolution.html"><a href="molecular-resolution.html#microbial-metagenotype-resolution"><i class="fa fa-check"></i>Resolution of microbial metagenotypes</a></li>
<li class="chapter" data-level="" data-path="molecular-resolution.html"><a href="molecular-resolution.html#envirotype-resolution"><i class="fa fa-check"></i>Resolution of envirotypes</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html"><i class="fa fa-check"></i><b>2.4</b> Spatiotemporal factors</a>
<ul>
<li class="chapter" data-level="" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html#spatial-factors"><i class="fa fa-check"></i>Spatial factors</a></li>
<li class="chapter" data-level="" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html#temporal-factors"><i class="fa fa-check"></i>Temporal factors</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html"><i class="fa fa-check"></i><b>2.5</b> Explanatory and response variables</a>
<ul>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#p-g-m-e"><i class="fa fa-check"></i>Phenotype as a product of genotype, metagenotype and envirotype</a></li>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#g-m-e"><i class="fa fa-check"></i>Genotype expression influenced by metagenotype and envirotype</a></li>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#m-g-e"><i class="fa fa-check"></i>Metagenotype as a product of genotype and envirotype</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II FIELD PROCEDURES</b></span></li>
<li class="chapter" data-level="3" data-path="about-fieldwork.html"><a href="about-fieldwork.html"><i class="fa fa-check"></i><b>3</b> About fieldwork</a></li>
<li class="chapter" data-level="4" data-path="fieldwork-sample-collection.html"><a href="fieldwork-sample-collection.html"><i class="fa fa-check"></i><b>4</b> Sample collection</a></li>
<li class="chapter" data-level="5" data-path="fieldwork-sample-preservation.html"><a href="fieldwork-sample-preservation.html"><i class="fa fa-check"></i><b>5</b> Sample preservation</a></li>
<li class="part"><span><b>III LABORATORY PROCEDURES</b></span></li>
<li class="chapter" data-level="6" data-path="about-labwork.html"><a href="about-labwork.html"><i class="fa fa-check"></i><b>6</b> About labwork</a>
<ul>
<li class="chapter" data-level="" data-path="about-labwork.html"><a href="about-labwork.html#laboratory-general-considerations"><i class="fa fa-check"></i>General considerations</a></li>
<li class="chapter" data-level="" data-path="about-labwork.html"><a href="about-labwork.html#laboratory-procedures"><i class="fa fa-check"></i>Procedures for generating multi-omic data</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html"><i class="fa fa-check"></i><b>7</b> DNA/RNA extraction</a>
<ul>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#sample-preprocessing"><i class="fa fa-check"></i>Sample preprocessing</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#chemical-isolation"><i class="fa fa-check"></i>Chemical isolation</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#physicochemical-isolation"><i class="fa fa-check"></i>Physicochemical isolation</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#available-protocols"><i class="fa fa-check"></i>Available protocols</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="protein-metabolite-extraction.html"><a href="protein-metabolite-extraction.html"><i class="fa fa-check"></i><b>8</b> Protein/metabolite extraction</a></li>
<li class="chapter" data-level="9" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html"><i class="fa fa-check"></i><b>9</b> Sequencing library preparation</a>
<ul>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#sequencing-strategies-platforms"><i class="fa fa-check"></i>Sequencing strategies and platforms</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#pcr-library"><i class="fa fa-check"></i>PCR-based vs. PCR-free library preparation</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#indices-multiplexing"><i class="fa fa-check"></i>Indices and multiplexing</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#unique-molecular-identifiers"><i class="fa fa-check"></i>Unique molecular identifiers (UMIs)</a></li>
<li class="chapter" data-level="9.1" data-path="library-meta-genomics.html"><a href="library-meta-genomics.html"><i class="fa fa-check"></i><b>9.1</b> Host genomics and microbial metagenomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-meta-genomics.html"><a href="library-meta-genomics.html#library-meta-genomics-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="library-host-transcriptomics.html"><a href="library-host-transcriptomics.html"><i class="fa fa-check"></i><b>9.2</b> Host transcriptomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-host-transcriptomics.html"><a href="library-host-transcriptomics.html#rrna-polya-depletion-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html"><i class="fa fa-check"></i><b>9.3</b> Microbial metatranscriptomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#capture-based-rrna-depletion"><i class="fa fa-check"></i>Capture-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#rnase-based-rrna-depletion"><i class="fa fa-check"></i>RNAse-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#cas9-based-rrna-depletion"><i class="fa fa-check"></i>CRISPR/Cas9-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#rrna-depletion-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV BIOINFORMATIC PROCEDURES</b></span></li>
<li class="chapter" data-level="10" data-path="about-bioinformatics.html"><a href="about-bioinformatics.html"><i class="fa fa-check"></i><b>10</b> About bioinformatics</a>
<ul>
<li class="chapter" data-level="10.1" data-path="prepare-shell.html"><a href="prepare-shell.html"><i class="fa fa-check"></i><b>10.1</b> Prepare your shell environment</a>
<ul>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#required-software"><i class="fa fa-check"></i>Required software</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-conda-miniconda"><i class="fa fa-check"></i>Install conda / miniconda</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-mamba"><i class="fa fa-check"></i>Install mamba (optional)</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#create-conda-environment"><i class="fa fa-check"></i>Create a conda environment</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#activate-the-holo-omics-conda-environment"><i class="fa fa-check"></i>Activate the holo-omics conda environment</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-software-in-conda-environment"><i class="fa fa-check"></i>Install software in conda environment</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="using-snakemake.html"><a href="using-snakemake.html"><i class="fa fa-check"></i><b>10.2</b> Using snakemake for workflow management</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html"><i class="fa fa-check"></i><b>11</b> Sequencing data preprocessing</a>
<ul>
<li class="chapter" data-level="" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html#preprocess-the-reads-using-fastp"><i class="fa fa-check"></i>Preprocess the reads using fastp</a></li>
<li class="chapter" data-level="" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html#splitting-host-and-non-host-data"><i class="fa fa-check"></i>Splitting host and non-host data</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="host-genomics-data-processing.html"><a href="host-genomics-data-processing.html"><i class="fa fa-check"></i><b>12</b> Host genomics (HG) data processing</a>
<ul>
<li class="chapter" data-level="12.1" data-path="host-reference-genome.html"><a href="host-reference-genome.html"><i class="fa fa-check"></i><b>12.1</b> Host reference genome</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-quality"><i class="fa fa-check"></i><b>12.1.1</b> Genome quality</a></li>
<li class="chapter" data-level="12.1.2" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-profile-analysis"><i class="fa fa-check"></i><b>12.1.2</b> Genome profile analysis</a></li>
<li class="chapter" data-level="12.1.3" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-assembly-hifiasm"><i class="fa fa-check"></i><b>12.1.3</b> Genome assembly using hifiasm</a></li>
<li class="chapter" data-level="12.1.4" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-assembly-evaluation"><i class="fa fa-check"></i><b>12.1.4</b> Assembly evaluation</a></li>
<li class="chapter" data-level="12.1.5" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-scaffolding"><i class="fa fa-check"></i><b>12.1.5</b> Assembly scaffolding</a></li>
<li class="chapter" data-level="12.1.6" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-final-evaluation"><i class="fa fa-check"></i><b>12.1.6</b> Final genome evaluation</a></li>
<li class="chapter" data-level="12.1.7" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-annotation"><i class="fa fa-check"></i><b>12.1.7</b> Reference genome annotation</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="host-resequencing.html"><a href="host-resequencing.html"><i class="fa fa-check"></i><b>12.2</b> Host genome resequencing</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="microbial-metagenomics-data-processing.html"><a href="microbial-metagenomics-data-processing.html"><i class="fa fa-check"></i><b>13</b> Microbial metagenomics (MG) data processing</a>
<ul>
<li class="chapter" data-level="13.1" data-path="reference-based.html"><a href="reference-based.html"><i class="fa fa-check"></i><b>13.1</b> Reference-based</a></li>
<li class="chapter" data-level="13.2" data-path="assembly-based.html"><a href="assembly-based.html"><i class="fa fa-check"></i><b>13.2</b> Assembly-based</a>
<ul>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#individual-assembly-based"><i class="fa fa-check"></i>Individual assembly-based</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#coassembly-based"><i class="fa fa-check"></i>Coassembly-based</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#assembly-gene-annotation"><i class="fa fa-check"></i>Gene annotation</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#assembly-read-mapping"><i class="fa fa-check"></i>Read mapping</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="genome-resolved.html"><a href="genome-resolved.html"><i class="fa fa-check"></i><b>13.3</b> Genome-resolved</a>
<ul>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-binning"><i class="fa fa-check"></i>Binning</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-refinement"><i class="fa fa-check"></i>Bin refinement</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-qc"><i class="fa fa-check"></i>Bin quality assessment</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-curation"><i class="fa fa-check"></i>Bin curation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-dereplication"><i class="fa fa-check"></i>Dereplication</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-taxonomy"><i class="fa fa-check"></i>Taxonomic annotation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-function"><i class="fa fa-check"></i>Functional annotation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-mapping"><i class="fa fa-check"></i>Read mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14" data-path="host-transcriptomics-data-processing.html"><a href="host-transcriptomics-data-processing.html"><i class="fa fa-check"></i><b>14</b> Host transcriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="14.1" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html"><i class="fa fa-check"></i><b>14.1</b> Reference-based host transcriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#quality-filtering"><i class="fa fa-check"></i>Quality-filtering</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#ribosomal-rna-removal"><i class="fa fa-check"></i>Ribosomal RNA removal</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#reference-genome-indexing"><i class="fa fa-check"></i>Reference genome indexing</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#read-mapping-against-reference-genome"><i class="fa fa-check"></i>Read mapping against reference genome</a></li>
</ul></li>
<li class="chapter" data-level="14.2" data-path="host-transcriptomics-data-processing-reference-free.html"><a href="host-transcriptomics-data-processing-reference-free.html"><i class="fa fa-check"></i><b>14.2</b> Reference-free host transcriptomics (HT) data processing</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="microbial-metatranscriptomics-data-processing.html"><a href="microbial-metatranscriptomics-data-processing.html"><i class="fa fa-check"></i><b>15</b> Microbial metatranscriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="15.1" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html"><i class="fa fa-check"></i><b>15.1</b> Reference-based microbial metatranscriptomics (MT) data processing</a>
<ul>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#quality-filtering-1"><i class="fa fa-check"></i>Quality filtering</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#ribosomal-rna-removal-1"><i class="fa fa-check"></i>Ribosomal RNA removal</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#host-genome-indexing"><i class="fa fa-check"></i>Host genome indexing</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#host-genome-mapping"><i class="fa fa-check"></i>Host genome mapping</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#generating-and-indexing-the-microbial-genome-catalogue"><i class="fa fa-check"></i>Generating and indexing the microbial genome catalogue</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#mapping-against-the-microbial-genome-catalogue"><i class="fa fa-check"></i>Mapping against the microbial genome catalogue</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#calculate-gene-counts"><i class="fa fa-check"></i>Calculate gene counts</a></li>
</ul></li>
<li class="chapter" data-level="15.2" data-path="microbial-metatranscriptomics-data-processing-reference-free.html"><a href="microbial-metatranscriptomics-data-processing-reference-free.html"><i class="fa fa-check"></i><b>15.2</b> Reference-free microbial metatranscriptomics (MT) data processing</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="host-proteomics-data-processing.html"><a href="host-proteomics-data-processing.html"><i class="fa fa-check"></i><b>16</b> Host proteomics (HP) data processing</a></li>
<li class="chapter" data-level="17" data-path="microbial-metaproteomics-data-processing.html"><a href="microbial-metaproteomics-data-processing.html"><i class="fa fa-check"></i><b>17</b> Microbial metaproteomics (MP) data processing</a></li>
<li class="part"><span><b>V STATISTICAL PROCEDURES</b></span></li>
<li class="chapter" data-level="18" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>18</b> About statistics</a>
<ul>
<li class="chapter" data-level="" data-path="about-statistics.html"><a href="about-statistics.html#a-step-by-step-approach"><i class="fa fa-check"></i>A step-by-step approach</a></li>
<li class="chapter" data-level="18.1" data-path="prepare-r.html"><a href="prepare-r.html"><i class="fa fa-check"></i><b>18.1</b> Prepare your R environment</a>
<ul>
<li class="chapter" data-level="" data-path="prepare-r.html"><a href="prepare-r.html#required-packages"><i class="fa fa-check"></i>Required packages</a></li>
<li class="chapter" data-level="" data-path="prepare-r.html"><a href="prepare-r.html#package-installation"><i class="fa fa-check"></i>Package installation</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="github-r.html"><a href="github-r.html"><i class="fa fa-check"></i><b>18.2</b> Create / clone a Github repository</a>
<ul>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#install-git-in-you-local-computer"><i class="fa fa-check"></i>Install git in you local computer</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#create-a-github-repository"><i class="fa fa-check"></i>Create a Github repository</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#create-a-version-control-project"><i class="fa fa-check"></i>Create a version-control project</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#set-up-rstudio-github-connection"><i class="fa fa-check"></i>Set-up RStudio-Github connection</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="19" data-path="single-omic-analyses.html"><a href="single-omic-analyses.html"><i class="fa fa-check"></i><b>19</b> Single omic analyses</a></li>
<li class="chapter" data-level="20" data-path="data-transformations.html"><a href="data-transformations.html"><i class="fa fa-check"></i><b>20</b> Data transformations</a>
<ul>
<li class="chapter" data-level="20.1" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html"><i class="fa fa-check"></i><b>20.1</b> Transformations to account for statistical assumptions</a>
<ul>
<li class="chapter" data-level="" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#transforming-data-to-meet-normality-assumption"><i class="fa fa-check"></i>Transforming data to meet normality assumption</a></li>
<li class="chapter" data-level="20.1.1" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#data-transformation-compositional"><i class="fa fa-check"></i><b>20.1.1</b> Transformations to account for compositional data</a></li>
<li class="chapter" data-level="20.1.2" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#data-transformation-scaling"><i class="fa fa-check"></i><b>20.1.2</b> Transformations to account for scaling</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="21" data-path="unsupervised-exploration.html"><a href="unsupervised-exploration.html"><i class="fa fa-check"></i><b>21</b> Unsupervised exploration</a>
<ul>
<li class="chapter" data-level="21.1" data-path="cluster-analysis.html"><a href="cluster-analysis.html"><i class="fa fa-check"></i><b>21.1</b> Cluster analysis</a>
<ul>
<li class="chapter" data-level="21.1.1" data-path="cluster-analysis.html"><a href="cluster-analysis.html#hierarchical-clustering"><i class="fa fa-check"></i><b>21.1.1</b> Hierarchical clustering</a></li>
<li class="chapter" data-level="21.1.2" data-path="cluster-analysis.html"><a href="cluster-analysis.html#disjoint-clustering"><i class="fa fa-check"></i><b>21.1.2</b> Disjoint clustering</a></li>
</ul></li>
<li class="chapter" data-level="21.2" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html"><i class="fa fa-check"></i><b>21.2</b> Dimension reduction and ordination</a>
<ul>
<li class="chapter" data-level="21.2.1" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#pca"><i class="fa fa-check"></i><b>21.2.1</b> Principal Component Analysis (PCA)</a></li>
<li class="chapter" data-level="21.2.2" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#pcoa"><i class="fa fa-check"></i><b>21.2.2</b> Principal Coordinate Analysis (PCoA)</a></li>
<li class="chapter" data-level="21.2.3" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#nmds"><i class="fa fa-check"></i><b>21.2.3</b> Non-metric Multidimensional Scaling (NMDS)</a></li>
<li class="chapter" data-level="21.2.4" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#t-sne"><i class="fa fa-check"></i><b>21.2.4</b> t-Distributed Stochastic Neighbour Embedding (t-SNE)</a></li>
<li class="chapter" data-level="21.2.5" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#umap"><i class="fa fa-check"></i><b>21.2.5</b> Uniform manifold approximation and projection (UMAP)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="22" data-path="supervised-analysis.html"><a href="supervised-analysis.html"><i class="fa fa-check"></i><b>22</b> Supervised analysis</a>
<ul>
<li class="chapter" data-level="22.1" data-path="regression-methods.html"><a href="regression-methods.html"><i class="fa fa-check"></i><b>22.1</b> Regression methods</a>
<ul>
<li class="chapter" data-level="22.1.1" data-path="regression-methods.html"><a href="regression-methods.html#permanova"><i class="fa fa-check"></i><b>22.1.1</b> PERMANOVA</a></li>
<li class="chapter" data-level="22.1.2" data-path="regression-methods.html"><a href="regression-methods.html#anosim"><i class="fa fa-check"></i><b>22.1.2</b> ANOSIM</a></li>
<li class="chapter" data-level="22.1.3" data-path="regression-methods.html"><a href="regression-methods.html#redundancy-analysis"><i class="fa fa-check"></i><b>22.1.3</b> Redundancy analysis (RDA)</a></li>
<li class="chapter" data-level="22.1.4" data-path="regression-methods.html"><a href="regression-methods.html#canonical-correspondence-analysis"><i class="fa fa-check"></i><b>22.1.4</b> Canonical Correspondence Analysis (CCA)</a></li>
<li class="chapter" data-level="22.1.5" data-path="regression-methods.html"><a href="regression-methods.html#generalised-linear-modelling"><i class="fa fa-check"></i><b>22.1.5</b> Generalised linear modelling (GLM)</a></li>
<li class="chapter" data-level="22.1.6" data-path="regression-methods.html"><a href="regression-methods.html#generalised-linear-mixed-modelling"><i class="fa fa-check"></i><b>22.1.6</b> Generalised linear mixed modelling (GLMM)</a></li>
</ul></li>
<li class="chapter" data-level="22.2" data-path="classification-methods.html"><a href="classification-methods.html"><i class="fa fa-check"></i><b>22.2</b> Classification methods</a>
<ul>
<li class="chapter" data-level="22.2.1" data-path="classification-methods.html"><a href="classification-methods.html#random-forests"><i class="fa fa-check"></i><b>22.2.1</b> Random Forests (RF)</a></li>
<li class="chapter" data-level="22.2.2" data-path="classification-methods.html"><a href="classification-methods.html#support-vector-machines"><i class="fa fa-check"></i><b>22.2.2</b> Support Vector Machines (SVM)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="23" data-path="multi-omic-integration.html"><a href="multi-omic-integration.html"><i class="fa fa-check"></i><b>23</b> Multi-omic integration</a></li>
<li class="chapter" data-level="24" data-path="multi-staged-omics-integration.html"><a href="multi-staged-omics-integration.html"><i class="fa fa-check"></i><b>24</b> Multi-staged omics integration</a></li>
<li class="chapter" data-level="25" data-path="meta-dimensional-omics-integration.html"><a href="meta-dimensional-omics-integration.html"><i class="fa fa-check"></i><b>25</b> Meta-dimensional omics integration</a>
<ul>
<li class="chapter" data-level="25.1" data-path="multi-staged-omics-integration-concatenation-based.html"><a href="multi-staged-omics-integration-concatenation-based.html"><i class="fa fa-check"></i><b>25.1</b> Concatenation-based integration</a></li>
<li class="chapter" data-level="25.2" data-path="multi-staged-omics-integration-transformation-based.html"><a href="multi-staged-omics-integration-transformation-based.html"><i class="fa fa-check"></i><b>25.2</b> Transformation-based integration</a></li>
<li class="chapter" data-level="25.3" data-path="multi-staged-omics-integration-model-based.html"><a href="multi-staged-omics-integration-model-based.html"><i class="fa fa-check"></i><b>25.3</b> Model-based integration</a></li>
</ul></li>
<li class="part"><span><b>VI RESOURCES</b></span></li>
<li class="chapter" data-level="26" data-path="useful-links.html"><a href="useful-links.html"><i class="fa fa-check"></i><b>26</b> Useful links</a>
<ul>
<li class="chapter" data-level="" data-path="useful-links.html"><a href="useful-links.html#links-data"><i class="fa fa-check"></i>Data access</a></li>
<li class="chapter" data-level="" data-path="useful-links.html"><a href="useful-links.html#links-documentation"><i class="fa fa-check"></i>Documentation</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>27</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A Practical Guide to Holo-Omics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="fieldwork-sample-preservation" class="section level1 hasAnchor" number="5">
<h1><span class="header-section-number">5</span> Sample preservation<a href="fieldwork-sample-preservation.html#fieldwork-sample-preservation" class="anchor-section" aria-label="Anchor link to header"></a></h1>
<p>Utilising liquid nitrogen for snap-freezing samples is widely regarded as the benchmark method for extracting pristine DNA from an unmodified microbial community. Nevertheless, the feasibility of promptly freezing samples while maintaining an unbroken cold chain isn’t always achievable, especially during field sampling missions. Consequently, dependable substitutes become imperative. This necessity has spurred the creation of various preservatives, enabling samples to be preserved at room temperature over prolonged durations. Nonetheless, research has unveiled that these preservatives can impart significant biases during the generation of diverse omic layers <span class="citation">[<a href="#ref-Perez-Losada2016-nc">15</a>]</span>. Therefore, maintaining uniformity in the selection of preservatives is of paramount importance to ensure consistency across analyses.</p>
<table>
<thead>
<tr class="header">
<th>Preservative</th>
<th>HG/MG</th>
<th>HT/MT</th>
<th>MP</th>
<th>ME</th>
<th>Reference</th>
</tr>
</thead>
<tbody>
<tr class="odd">
<td>Snap frozen</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td><span class="citation">[<a href="#ref-De_Spiegeleer2020-qc">16</a>]</span></td>
</tr>
<tr class="even">
<td>RNAlater</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Van_Eijsden2013-dx">17</a>]</span></td>
</tr>
<tr class="odd">
<td>DNA/RNA Shield</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Schweighardt2015-xk">18</a>]</span></td>
</tr>
<tr class="even">
<td>OMNIgene GUT</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Wang2018-fs">19</a>]</span></td>
</tr>
<tr class="odd">
<td>Tris-EDTA Buffer</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Barra2015-nv">20</a>]</span></td>
</tr>
<tr class="even">
<td>Guanidine thiocyanate</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Weidner2022-da">21</a>]</span></td>
</tr>
<tr class="odd">
<td>TRIzol</td>
<td>Yes</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Simoes2013-xh">22</a>]</span></td>
</tr>
<tr class="even">
<td>Protease inhibitors</td>
<td>No</td>
<td>No</td>
<td>Yes</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Ryan2017-pl">23</a>]</span></td>
</tr>
<tr class="odd">
<td>FTA Cards</td>
<td>Yes</td>
<td>Yes</td>
<td>No</td>
<td>Yes</td>
<td><span class="citation">[<a href="#ref-Bolt_Botnen2023-ql">24</a>]</span></td>
</tr>
<tr class="even">
<td>Methanol</td>
<td>No</td>
<td>No</td>
<td>No</td>
<td>Yes</td>
<td><span class="citation">[<a href="#ref-Straughen2023-ia">25</a>]</span></td>
</tr>
<tr class="odd">
<td>Ammonium bicarbonate</td>
<td>No</td>
<td>No</td>
<td>Yes</td>
<td>No</td>
<td><span class="citation">[<a href="#ref-Hedges2013-ip">26</a>]</span></td>
</tr>
</tbody>
</table>
<p>Numerous preservatives necessitate removal before extraction, such as ethanol, NAP buffer, and RNAlater. However, this removal process can inadvertently eliminate non-pelleting entities, including bacteriophages and other viruses. Conversely, certain preservatives also serve as lysis buffers, exemplified by Zymo’s DNA/RNA Shield. Notably advantageous, these buffers directly participate in DNA extraction. Although these preservatives do not stabilise DNA within cells, they initiate cellular degradation while stabilising DNA in the matrix. Maintaining the recommended material-to-buffer ratio is imperative across all these buffers to ensure optimal outcomes. Overloading with biological material can negate the beneficial effects of the buffers.</p>
<p>Avoiding freeze-thaw cycles is ideal, as they are recognised sources of DNA degradation and variations in microbial community composition, especially when samples are repetitively thawed <span class="citation">[<a href="#ref-Cuthbertson2015-uy">27</a>]</span>. Opting for small aliquots tailored to the extraction protocol during sample collection, rather than bulk collection, facilitates thawing only the sample intended for processing. This practice sidesteps detrimental thaw-freeze cycles and diminishes the risk of cross-contamination from other samples.</p>
<p>Furthermore, the biological and chemical characteristics of molecules (e.g., DNA, RNA, proteins, metabolites) used in omic data generation must be acknowledged. Host DNA’s abundance and stability render HG less sensitive. Conversely, MG warrants more cautious handling due to potential microbial community fluctuations post-sampling, unless biochemical reactions are halted. HT and MT demand even swifter preservation to capture representative gene expression patterns. Lastly, metabolites exhibit diverse chemical properties, ranging from stable steroids to highly volatile short-chain fatty acids. Thus, judicious selection of appropriate preservatives becomes paramount when generating multiple omic layers. This decision involves determining whether omic data will be sourced from a single biological sample, necessitating a universal preservative, or multiple samples, each potentially requiring a distinct preservative.</p>
<p>Importantly, the diverse physicochemical properties of samples mandate that collection and storage methods validated for one sample type cannot be universally assumed optimal for others. Therefore, preliminary optimisation tests are prudent, and methodological consistency emerges as a prerequisite for the production of dependable omic data.</p>
<div class="authorbox">
<p>Contents of this section were created by <a href="index.html#antton-alberdi">Antton Alberdi</a>, <a href="index.html#ostaizka-aizpurua">Ostaizka Aizpurua</a> and <a href="index.html#jacob-rasmussen">Jacob A Rasmussen</a>.</p>
</div>

</div>



<h3> References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-Perez-Losada2016-nc" class="csl-entry">
15. Pérez-Losada M, Crandall KA, Freishtat RJ. Two sampling methods yield distinct microbial signatures in the nasopharynges of asthmatic children. Microbiome. 2016;4:25.
</div>
<div id="ref-De_Spiegeleer2020-qc" class="csl-entry">
16. De Spiegeleer M, De Graeve M, Huysman S, Vanderbeke A, Van Meulebroek L, Vanhaecke L. Impact of storage conditions on the human stool metabolome and lipidome: Preserving the most accurate fingerprint. Anal Chim Acta. 2020;1108:79–88.
</div>
<div id="ref-Van_Eijsden2013-dx" class="csl-entry">
17. Eijsden RGE van, Stassen C, Daenen L, Van Mulders SE, Bapat PM, Siewers V, et al. A universal fixation method based on quaternary ammonium salts (<span>RNAlater</span>) for omics-technologies: Saccharomyces cerevisiae as a case study. Biotechnol Lett. 2013;35:891–900.
</div>
<div id="ref-Schweighardt2015-xk" class="csl-entry">
18. Schweighardt AJ, Tate CM, Scott KA, Harper KA, Robertson JM. Evaluation of commercial kits for dual extraction of <span>DNA</span> and <span>RNA</span> from human body fluids. J Forensic Sci. 2015;60:157–65.
</div>
<div id="ref-Wang2018-fs" class="csl-entry">
19. Wang Z, Zolnik CP, Qiu Y, Usyk M, Wang T, Strickler HD, et al. Comparison of fecal collection methods for microbiome and metabolomics studies. Front Cell Infect Microbiol. 2018;8:301.
</div>
<div id="ref-Barra2015-nv" class="csl-entry">
20. Barra GB, Santa Rita TH, Almeida Vasques J de, Chianca CF, Nery LFA, Santana Soares Costa S. <span class="nocase">EDTA-mediated</span> inhibition of <span>DNases</span> protects circulating cell-free <span>DNA</span> from ex vivo degradation in blood samples. Clin Biochem. 2015;48:976–81.
</div>
<div id="ref-Weidner2022-da" class="csl-entry">
21. Weidner L, Laner-Plamberger S, Horner D, Pistorius C, Jurkin J, Karbiener M, et al. Sample buffer containing <span>Guanidine-Hydrochloride</span> combines biological safety and <span>RNA</span> preservation for <span>SARS-CoV-2</span> molecular diagnostics. Diagnostics (Basel). 2022;12.
</div>
<div id="ref-Simoes2013-xh" class="csl-entry">
22. Simões AES, Pereira DM, Amaral JD, Nunes AF, Gomes SE, Rodrigues PM, et al. Efficient recovery of proteins from multiple source samples after trizol or <span class="nocase">trizolLS</span> <span>RNA</span> extraction and long-term storage. BMC Genomics. 2013;14:1–15.
</div>
<div id="ref-Ryan2017-pl" class="csl-entry">
23. Ryan BJ, Henehan GT. Avoiding proteolysis during protein purification. Methods Mol Biol. 2017;1485:53–69.
</div>
<div id="ref-Bolt_Botnen2023-ql" class="csl-entry">
24. Bolt Botnen A, Bjørnsen MB, Alberdi A, Gilbert MTP, Aizpurua O. A simplified protocol for <span>DNA</span> extraction from <span>FTA</span> cards for faecal microbiome studies. Heliyon. 2023;e12861.
</div>
<div id="ref-Straughen2023-ia" class="csl-entry">
25. Straughen JK, Sitarik AR, Jones AD, Li J, Allo G, Salafia C, et al. Comparison of methanol fixation versus cryopreservation of the placenta for metabolomics analysis. Sci Rep. 2023;13:4063.
</div>
<div id="ref-Hedges2013-ip" class="csl-entry">
26. Hedges JB, Vahidi S, Yue X, Konermann L. Effects of ammonium bicarbonate on the electrospray mass spectra of proteins: Evidence for bubble-induced unfolding. Anal Chem. 2013;85:6469–76.
</div>
<div id="ref-Cuthbertson2015-uy" class="csl-entry">
27. Cuthbertson L, Rogers GB, Walker AW, Oliver A, Hoffman LR, Carroll MP, et al. Implications of multiple freeze-thawing on respiratory samples for culture-independent analyses. J Cyst Fibros. 2015;14:464–7.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="fieldwork-sample-collection.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="about-labwork.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

</body>

</html>
