<!DOCTYPE html>
<html lang="" xml:lang="">
<head>

  <meta charset="utf-8" />
  <meta http-equiv="X-UA-Compatible" content="IE=edge" />
  <title>25.1 Concatenation-based integration | A Practical Guide to Holo-Omics</title>
  <meta name="description" content="25.1 Concatenation-based integration | A Practical Guide to Holo-Omics" />
  <meta name="generator" content="bookdown 0.34 and GitBook 2.6.7" />

  <meta property="og:title" content="25.1 Concatenation-based integration | A Practical Guide to Holo-Omics" />
  <meta property="og:type" content="book" />
  
  
  

  <meta name="twitter:card" content="summary" />
  <meta name="twitter:title" content="25.1 Concatenation-based integration | A Practical Guide to Holo-Omics" />
  
  
  

<meta name="author" content="The Center for Evolutionary Hologenomics" />



  <meta name="viewport" content="width=device-width, initial-scale=1" />
  <meta name="apple-mobile-web-app-capable" content="yes" />
  <meta name="apple-mobile-web-app-status-bar-style" content="black" />
  
  
<link rel="prev" href="meta-dimensional-omics-integration.html"/>
<link rel="next" href="multi-staged-omics-integration-transformation-based.html"/>
<script src="libs/jquery-3.6.0/jquery-3.6.0.min.js"></script>
<script src="https://cdn.jsdelivr.net/npm/fuse.js@6.4.6/dist/fuse.min.js"></script>
<link href="libs/gitbook-2.6.7/css/style.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-table.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-bookdown.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-highlight.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-search.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-fontsettings.css" rel="stylesheet" />
<link href="libs/gitbook-2.6.7/css/plugin-clipboard.css" rel="stylesheet" />









<link href="libs/anchor-sections-1.1.0/anchor-sections.css" rel="stylesheet" />
<link href="libs/anchor-sections-1.1.0/anchor-sections-hash.css" rel="stylesheet" />
<script src="libs/anchor-sections-1.1.0/anchor-sections.js"></script>
<!-- Google tag (gtag.js) -->
<script async src="https://www.googletagmanager.com/gtag/js?id=G-45DK1TCMQC"></script>
<script>
  window.dataLayer = window.dataLayer || [];
  function gtag(){dataLayer.push(arguments);}
  gtag('js', new Date());

  gtag('config', 'G-45DK1TCMQC');
</script>


<style type="text/css">
pre > code.sourceCode { white-space: pre; position: relative; }
pre > code.sourceCode > span { display: inline-block; line-height: 1.25; }
pre > code.sourceCode > span:empty { height: 1.2em; }
.sourceCode { overflow: visible; }
code.sourceCode > span { color: inherit; text-decoration: inherit; }
pre.sourceCode { margin: 0; }
@media screen {
div.sourceCode { overflow: auto; }
}
@media print {
pre > code.sourceCode { white-space: pre-wrap; }
pre > code.sourceCode > span { text-indent: -5em; padding-left: 5em; }
}
pre.numberSource code
  { counter-reset: source-line 0; }
pre.numberSource code > span
  { position: relative; left: -4em; counter-increment: source-line; }
pre.numberSource code > span > a:first-child::before
  { content: counter(source-line);
    position: relative; left: -1em; text-align: right; vertical-align: baseline;
    border: none; display: inline-block;
    -webkit-touch-callout: none; -webkit-user-select: none;
    -khtml-user-select: none; -moz-user-select: none;
    -ms-user-select: none; user-select: none;
    padding: 0 4px; width: 4em;
    color: #aaaaaa;
  }
pre.numberSource { margin-left: 3em; border-left: 1px solid #aaaaaa;  padding-left: 4px; }
div.sourceCode
  {  background-color: #f8f8f8; }
@media screen {
pre > code.sourceCode > span > a:first-child::before { text-decoration: underline; }
}
code span.al { color: #ef2929; } /* Alert */
code span.an { color: #8f5902; font-weight: bold; font-style: italic; } /* Annotation */
code span.at { color: #204a87; } /* Attribute */
code span.bn { color: #0000cf; } /* BaseN */
code span.cf { color: #204a87; font-weight: bold; } /* ControlFlow */
code span.ch { color: #4e9a06; } /* Char */
code span.cn { color: #8f5902; } /* Constant */
code span.co { color: #8f5902; font-style: italic; } /* Comment */
code span.cv { color: #8f5902; font-weight: bold; font-style: italic; } /* CommentVar */
code span.do { color: #8f5902; font-weight: bold; font-style: italic; } /* Documentation */
code span.dt { color: #204a87; } /* DataType */
code span.dv { color: #0000cf; } /* DecVal */
code span.er { color: #a40000; font-weight: bold; } /* Error */
code span.ex { } /* Extension */
code span.fl { color: #0000cf; } /* Float */
code span.fu { color: #204a87; font-weight: bold; } /* Function */
code span.im { } /* Import */
code span.in { color: #8f5902; font-weight: bold; font-style: italic; } /* Information */
code span.kw { color: #204a87; font-weight: bold; } /* Keyword */
code span.op { color: #ce5c00; font-weight: bold; } /* Operator */
code span.ot { color: #8f5902; } /* Other */
code span.pp { color: #8f5902; font-style: italic; } /* Preprocessor */
code span.sc { color: #ce5c00; font-weight: bold; } /* SpecialChar */
code span.ss { color: #4e9a06; } /* SpecialString */
code span.st { color: #4e9a06; } /* String */
code span.va { color: #000000; } /* Variable */
code span.vs { color: #4e9a06; } /* VerbatimString */
code span.wa { color: #8f5902; font-weight: bold; font-style: italic; } /* Warning */
</style>

<style type="text/css">
  
  div.hanging-indent{margin-left: 1.5em; text-indent: -1.5em;}
</style>
<style type="text/css">
/* Used with Pandoc 2.11+ new --citeproc when CSL is used */
div.csl-bib-body { }
div.csl-entry {
  clear: both;
}
.hanging div.csl-entry {
  margin-left:2em;
  text-indent:-2em;
}
div.csl-left-margin {
  min-width:2em;
  float:left;
}
div.csl-right-inline {
  margin-left:2em;
  padding-left:1em;
}
div.csl-indent {
  margin-left: 2em;
}
</style>

<link rel="stylesheet" href="style.css" type="text/css" />
</head>

<body>



  <div class="book without-animation with-summary font-size-2 font-family-1" data-basepath=".">

    <div class="book-summary">
      <nav role="navigation">

<ul class="summary">
<li class="chapter" data-level="" data-path="index.html"><a href="index.html"><i class="fa fa-check"></i>About this guidebook</a>
<ul>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#contents"><i class="fa fa-check"></i>Contents</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#protocols-exercises-tutorials"><i class="fa fa-check"></i>Protocols, exercises and tutorials</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#datasets"><i class="fa fa-check"></i>Data sets</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#about-the-authors"><i class="fa fa-check"></i>About the authors</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#how-to-cite-this-work"><i class="fa fa-check"></i>How to cite this work</a></li>
<li class="chapter" data-level="" data-path="index.html"><a href="index.html#acknowledgement"><i class="fa fa-check"></i>Acknowledgement</a></li>
</ul></li>
<li class="part"><span><b>I INTRODUCTION</b></span></li>
<li class="chapter" data-level="1" data-path="holo-omics.html"><a href="holo-omics.html"><i class="fa fa-check"></i><b>1</b> Introduction to holo-omics</a>
<ul>
<li class="chapter" data-level="" data-path="holo-omics.html"><a href="holo-omics.html#why-holo-omics"><i class="fa fa-check"></i>Why do we need holo-omics?</a></li>
<li class="chapter" data-level="" data-path="holo-omics.html"><a href="holo-omics.html#what-is-holo-omics"><i class="fa fa-check"></i>What is holo-omics?</a></li>
<li class="chapter" data-level="1.1" data-path="omic-layers.html"><a href="omic-layers.html"><i class="fa fa-check"></i><b>1.1</b> Omic layers</a>
<ul>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-genomics"><i class="fa fa-check"></i>Host genomics (HG)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-transcriptomics"><i class="fa fa-check"></i>Host transcriptomics (HT)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metagenomics"><i class="fa fa-check"></i>Microbial metagenomics (MG)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metatranscriptomics"><i class="fa fa-check"></i>Microbial metatranscriptomics (MT)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#host-proteomics"><i class="fa fa-check"></i>Host proteomics (HP)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#microbial-metaproteomics"><i class="fa fa-check"></i>Microbial metaproteomics (MP)</a></li>
<li class="chapter" data-level="" data-path="omic-layers.html"><a href="omic-layers.html#meta-metabolomics"><i class="fa fa-check"></i>(Meta)metabolomics (ME)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="2" data-path="study-design-considerations.html"><a href="study-design-considerations.html"><i class="fa fa-check"></i><b>2</b> Study design considerations</a>
<ul>
<li class="chapter" data-level="2.1" data-path="hologenomic-complexity.html"><a href="hologenomic-complexity.html"><i class="fa fa-check"></i><b>2.1</b> Hologenomic complexity</a></li>
<li class="chapter" data-level="2.2" data-path="control-of-variables.html"><a href="control-of-variables.html"><i class="fa fa-check"></i><b>2.2</b> Control of variables</a>
<ul>
<li class="chapter" data-level="2.2.1" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-host-genomes"><i class="fa fa-check"></i><b>2.2.1</b> Controlling host genomes</a></li>
<li class="chapter" data-level="2.2.2" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-microbial-metagenomes"><i class="fa fa-check"></i><b>2.2.2</b> Controlling microbial metagenomes</a></li>
<li class="chapter" data-level="2.2.3" data-path="control-of-variables.html"><a href="control-of-variables.html#controlling-environment"><i class="fa fa-check"></i><b>2.2.3</b> Controlling the environment</a></li>
</ul></li>
<li class="chapter" data-level="2.3" data-path="molecular-resolution.html"><a href="molecular-resolution.html"><i class="fa fa-check"></i><b>2.3</b> Molecular resolution</a>
<ul>
<li class="chapter" data-level="2.3.1" data-path="molecular-resolution.html"><a href="molecular-resolution.html#host-genotype-resolution"><i class="fa fa-check"></i><b>2.3.1</b> Resolution of host genotypes</a></li>
<li class="chapter" data-level="" data-path="molecular-resolution.html"><a href="molecular-resolution.html#microbial-metagenotype-resolution"><i class="fa fa-check"></i>Resolution of microbial metagenotypes</a></li>
<li class="chapter" data-level="" data-path="molecular-resolution.html"><a href="molecular-resolution.html#envirotype-resolution"><i class="fa fa-check"></i>Resolution of envirotypes</a></li>
</ul></li>
<li class="chapter" data-level="2.4" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html"><i class="fa fa-check"></i><b>2.4</b> Spatiotemporal factors</a>
<ul>
<li class="chapter" data-level="" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html#spatial-factors"><i class="fa fa-check"></i>Spatial factors</a></li>
<li class="chapter" data-level="" data-path="spatiotemporal-factors.html"><a href="spatiotemporal-factors.html#temporal-factors"><i class="fa fa-check"></i>Temporal factors</a></li>
</ul></li>
<li class="chapter" data-level="2.5" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html"><i class="fa fa-check"></i><b>2.5</b> Explanatory and response variables</a>
<ul>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#p-g-m-e"><i class="fa fa-check"></i>Phenotype as a product of genotype, metagenotype and envirotype</a></li>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#g-m-e"><i class="fa fa-check"></i>Genotype expression influenced by metagenotype and envirotype</a></li>
<li class="chapter" data-level="" data-path="explanatory-and-response-variables.html"><a href="explanatory-and-response-variables.html#m-g-e"><i class="fa fa-check"></i>Metagenotype as a product of genotype and envirotype</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>II FIELD PROCEDURES</b></span></li>
<li class="chapter" data-level="3" data-path="about-fieldwork.html"><a href="about-fieldwork.html"><i class="fa fa-check"></i><b>3</b> About fieldwork</a></li>
<li class="chapter" data-level="4" data-path="fieldwork-sample-collection.html"><a href="fieldwork-sample-collection.html"><i class="fa fa-check"></i><b>4</b> Sample collection</a></li>
<li class="chapter" data-level="5" data-path="fieldwork-sample-preservation.html"><a href="fieldwork-sample-preservation.html"><i class="fa fa-check"></i><b>5</b> Sample preservation</a></li>
<li class="part"><span><b>III LABORATORY PROCEDURES</b></span></li>
<li class="chapter" data-level="6" data-path="about-labwork.html"><a href="about-labwork.html"><i class="fa fa-check"></i><b>6</b> About labwork</a>
<ul>
<li class="chapter" data-level="" data-path="about-labwork.html"><a href="about-labwork.html#laboratory-general-considerations"><i class="fa fa-check"></i>General considerations</a></li>
<li class="chapter" data-level="" data-path="about-labwork.html"><a href="about-labwork.html#laboratory-procedures"><i class="fa fa-check"></i>Procedures for generating multi-omic data</a></li>
</ul></li>
<li class="chapter" data-level="7" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html"><i class="fa fa-check"></i><b>7</b> DNA/RNA extraction</a>
<ul>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#sample-preprocessing"><i class="fa fa-check"></i>Sample preprocessing</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#chemical-isolation"><i class="fa fa-check"></i>Chemical isolation</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#physicochemical-isolation"><i class="fa fa-check"></i>Physicochemical isolation</a></li>
<li class="chapter" data-level="" data-path="dna-rna-extraction.html"><a href="dna-rna-extraction.html#available-protocols"><i class="fa fa-check"></i>Available protocols</a></li>
</ul></li>
<li class="chapter" data-level="8" data-path="protein-metabolite-extraction.html"><a href="protein-metabolite-extraction.html"><i class="fa fa-check"></i><b>8</b> Protein/metabolite extraction</a></li>
<li class="chapter" data-level="9" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html"><i class="fa fa-check"></i><b>9</b> Sequencing library preparation</a>
<ul>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#sequencing-strategies-platforms"><i class="fa fa-check"></i>Sequencing strategies and platforms</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#pcr-library"><i class="fa fa-check"></i>PCR-based vs. PCR-free library preparation</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#indices-multiplexing"><i class="fa fa-check"></i>Indices and multiplexing</a></li>
<li class="chapter" data-level="" data-path="sequencing-library-preparation.html"><a href="sequencing-library-preparation.html#unique-molecular-identifiers"><i class="fa fa-check"></i>Unique molecular identifiers (UMIs)</a></li>
<li class="chapter" data-level="9.1" data-path="library-meta-genomics.html"><a href="library-meta-genomics.html"><i class="fa fa-check"></i><b>9.1</b> Host genomics and microbial metagenomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-meta-genomics.html"><a href="library-meta-genomics.html#library-meta-genomics-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
<li class="chapter" data-level="9.2" data-path="library-host-transcriptomics.html"><a href="library-host-transcriptomics.html"><i class="fa fa-check"></i><b>9.2</b> Host transcriptomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-host-transcriptomics.html"><a href="library-host-transcriptomics.html#rrna-polya-depletion-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
<li class="chapter" data-level="9.3" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html"><i class="fa fa-check"></i><b>9.3</b> Microbial metatranscriptomics</a>
<ul>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#capture-based-rrna-depletion"><i class="fa fa-check"></i>Capture-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#rnase-based-rrna-depletion"><i class="fa fa-check"></i>RNAse-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#cas9-based-rrna-depletion"><i class="fa fa-check"></i>CRISPR/Cas9-based rRNA depletion</a></li>
<li class="chapter" data-level="" data-path="library-microbial-metatranscriptomics.html"><a href="library-microbial-metatranscriptomics.html#rrna-depletion-protocols"><i class="fa fa-check"></i>List of available protocols</a></li>
</ul></li>
</ul></li>
<li class="part"><span><b>IV BIOINFORMATIC PROCEDURES</b></span></li>
<li class="chapter" data-level="10" data-path="about-bioinformatics.html"><a href="about-bioinformatics.html"><i class="fa fa-check"></i><b>10</b> About bioinformatics</a>
<ul>
<li class="chapter" data-level="10.1" data-path="prepare-shell.html"><a href="prepare-shell.html"><i class="fa fa-check"></i><b>10.1</b> Prepare your shell environment</a>
<ul>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#required-software"><i class="fa fa-check"></i>Required software</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-conda-miniconda"><i class="fa fa-check"></i>Install conda / miniconda</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-mamba"><i class="fa fa-check"></i>Install mamba (optional)</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#create-conda-environment"><i class="fa fa-check"></i>Create a conda environment</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#activate-the-holo-omics-conda-environment"><i class="fa fa-check"></i>Activate the holo-omics conda environment</a></li>
<li class="chapter" data-level="" data-path="prepare-shell.html"><a href="prepare-shell.html#install-software-in-conda-environment"><i class="fa fa-check"></i>Install software in conda environment</a></li>
</ul></li>
<li class="chapter" data-level="10.2" data-path="using-snakemake.html"><a href="using-snakemake.html"><i class="fa fa-check"></i><b>10.2</b> Using snakemake for workflow management</a></li>
</ul></li>
<li class="chapter" data-level="11" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html"><i class="fa fa-check"></i><b>11</b> Sequencing data preprocessing</a>
<ul>
<li class="chapter" data-level="" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html#preprocess-the-reads-using-fastp"><i class="fa fa-check"></i>Preprocess the reads using fastp</a></li>
<li class="chapter" data-level="" data-path="sequencing-data-preprocessing.html"><a href="sequencing-data-preprocessing.html#splitting-host-and-non-host-data"><i class="fa fa-check"></i>Splitting host and non-host data</a></li>
</ul></li>
<li class="chapter" data-level="12" data-path="host-genomics-data-processing.html"><a href="host-genomics-data-processing.html"><i class="fa fa-check"></i><b>12</b> Host genomics (HG) data processing</a>
<ul>
<li class="chapter" data-level="12.1" data-path="host-reference-genome.html"><a href="host-reference-genome.html"><i class="fa fa-check"></i><b>12.1</b> Host reference genome</a>
<ul>
<li class="chapter" data-level="12.1.1" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-quality"><i class="fa fa-check"></i><b>12.1.1</b> Genome quality</a></li>
<li class="chapter" data-level="12.1.2" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-profile-analysis"><i class="fa fa-check"></i><b>12.1.2</b> Genome profile analysis</a></li>
<li class="chapter" data-level="12.1.3" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-assembly-hifiasm"><i class="fa fa-check"></i><b>12.1.3</b> Genome assembly using hifiasm</a></li>
<li class="chapter" data-level="12.1.4" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-assembly-evaluation"><i class="fa fa-check"></i><b>12.1.4</b> Assembly evaluation</a></li>
<li class="chapter" data-level="12.1.5" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-scaffolding"><i class="fa fa-check"></i><b>12.1.5</b> Assembly scaffolding</a></li>
<li class="chapter" data-level="12.1.6" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-final-evaluation"><i class="fa fa-check"></i><b>12.1.6</b> Final genome evaluation</a></li>
<li class="chapter" data-level="12.1.7" data-path="host-reference-genome.html"><a href="host-reference-genome.html#host-genome-annotation"><i class="fa fa-check"></i><b>12.1.7</b> Reference genome annotation</a></li>
</ul></li>
<li class="chapter" data-level="12.2" data-path="host-resequencing.html"><a href="host-resequencing.html"><i class="fa fa-check"></i><b>12.2</b> Host genome resequencing</a></li>
</ul></li>
<li class="chapter" data-level="13" data-path="microbial-metagenomics-data-processing.html"><a href="microbial-metagenomics-data-processing.html"><i class="fa fa-check"></i><b>13</b> Microbial metagenomics (MG) data processing</a>
<ul>
<li class="chapter" data-level="13.1" data-path="reference-based.html"><a href="reference-based.html"><i class="fa fa-check"></i><b>13.1</b> Reference-based</a></li>
<li class="chapter" data-level="13.2" data-path="assembly-based.html"><a href="assembly-based.html"><i class="fa fa-check"></i><b>13.2</b> Assembly-based</a>
<ul>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#individual-assembly-based"><i class="fa fa-check"></i>Individual assembly-based</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#coassembly-based"><i class="fa fa-check"></i>Coassembly-based</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#assembly-gene-annotation"><i class="fa fa-check"></i>Gene annotation</a></li>
<li class="chapter" data-level="" data-path="assembly-based.html"><a href="assembly-based.html#assembly-read-mapping"><i class="fa fa-check"></i>Read mapping</a></li>
</ul></li>
<li class="chapter" data-level="13.3" data-path="genome-resolved.html"><a href="genome-resolved.html"><i class="fa fa-check"></i><b>13.3</b> Genome-resolved</a>
<ul>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-binning"><i class="fa fa-check"></i>Binning</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-refinement"><i class="fa fa-check"></i>Bin refinement</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-qc"><i class="fa fa-check"></i>Bin quality assessment</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-curation"><i class="fa fa-check"></i>Bin curation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-dereplication"><i class="fa fa-check"></i>Dereplication</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-taxonomy"><i class="fa fa-check"></i>Taxonomic annotation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-function"><i class="fa fa-check"></i>Functional annotation</a></li>
<li class="chapter" data-level="" data-path="genome-resolved.html"><a href="genome-resolved.html#genome-resolved-mapping"><i class="fa fa-check"></i>Read mapping</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="14" data-path="host-transcriptomics-data-processing.html"><a href="host-transcriptomics-data-processing.html"><i class="fa fa-check"></i><b>14</b> Host transcriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="14.1" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html"><i class="fa fa-check"></i><b>14.1</b> Reference-based host transcriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#quality-filtering"><i class="fa fa-check"></i>Quality-filtering</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#ribosomal-rna-removal"><i class="fa fa-check"></i>Ribosomal RNA removal</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#reference-genome-indexing"><i class="fa fa-check"></i>Reference genome indexing</a></li>
<li class="chapter" data-level="" data-path="host-transcriptomics-data-processing-reference-based.html"><a href="host-transcriptomics-data-processing-reference-based.html#read-mapping-against-reference-genome"><i class="fa fa-check"></i>Read mapping against reference genome</a></li>
</ul></li>
<li class="chapter" data-level="14.2" data-path="host-transcriptomics-data-processing-reference-free.html"><a href="host-transcriptomics-data-processing-reference-free.html"><i class="fa fa-check"></i><b>14.2</b> Reference-free host transcriptomics (HT) data processing</a></li>
</ul></li>
<li class="chapter" data-level="15" data-path="microbial-metatranscriptomics-data-processing.html"><a href="microbial-metatranscriptomics-data-processing.html"><i class="fa fa-check"></i><b>15</b> Microbial metatranscriptomics (HT) data processing</a>
<ul>
<li class="chapter" data-level="15.1" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html"><i class="fa fa-check"></i><b>15.1</b> Reference-based microbial metatranscriptomics (MT) data processing</a>
<ul>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#quality-filtering-1"><i class="fa fa-check"></i>Quality filtering</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#ribosomal-rna-removal-1"><i class="fa fa-check"></i>Ribosomal RNA removal</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#host-genome-indexing"><i class="fa fa-check"></i>Host genome indexing</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#host-genome-mapping"><i class="fa fa-check"></i>Host genome mapping</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#generating-and-indexing-the-microbial-genome-catalogue"><i class="fa fa-check"></i>Generating and indexing the microbial genome catalogue</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#mapping-against-the-microbial-genome-catalogue"><i class="fa fa-check"></i>Mapping against the microbial genome catalogue</a></li>
<li class="chapter" data-level="" data-path="microbial-metatranscriptomics-data-processing-reference-based.html"><a href="microbial-metatranscriptomics-data-processing-reference-based.html#calculate-gene-counts"><i class="fa fa-check"></i>Calculate gene counts</a></li>
</ul></li>
<li class="chapter" data-level="15.2" data-path="microbial-metatranscriptomics-data-processing-reference-free.html"><a href="microbial-metatranscriptomics-data-processing-reference-free.html"><i class="fa fa-check"></i><b>15.2</b> Reference-free microbial metatranscriptomics (MT) data processing</a></li>
</ul></li>
<li class="chapter" data-level="16" data-path="host-proteomics-data-processing.html"><a href="host-proteomics-data-processing.html"><i class="fa fa-check"></i><b>16</b> Host proteomics (HP) data processing</a></li>
<li class="chapter" data-level="17" data-path="microbial-metaproteomics-data-processing.html"><a href="microbial-metaproteomics-data-processing.html"><i class="fa fa-check"></i><b>17</b> Microbial metaproteomics (MP) data processing</a></li>
<li class="part"><span><b>V STATISTICAL PROCEDURES</b></span></li>
<li class="chapter" data-level="18" data-path="about-statistics.html"><a href="about-statistics.html"><i class="fa fa-check"></i><b>18</b> About statistics</a>
<ul>
<li class="chapter" data-level="" data-path="about-statistics.html"><a href="about-statistics.html#a-step-by-step-approach"><i class="fa fa-check"></i>A step-by-step approach</a></li>
<li class="chapter" data-level="18.1" data-path="prepare-r.html"><a href="prepare-r.html"><i class="fa fa-check"></i><b>18.1</b> Prepare your R environment</a>
<ul>
<li class="chapter" data-level="" data-path="prepare-r.html"><a href="prepare-r.html#required-packages"><i class="fa fa-check"></i>Required packages</a></li>
<li class="chapter" data-level="" data-path="prepare-r.html"><a href="prepare-r.html#package-installation"><i class="fa fa-check"></i>Package installation</a></li>
</ul></li>
<li class="chapter" data-level="18.2" data-path="github-r.html"><a href="github-r.html"><i class="fa fa-check"></i><b>18.2</b> Create / clone a Github repository</a>
<ul>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#install-git-in-you-local-computer"><i class="fa fa-check"></i>Install git in you local computer</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#create-a-github-repository"><i class="fa fa-check"></i>Create a Github repository</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#create-a-version-control-project"><i class="fa fa-check"></i>Create a version-control project</a></li>
<li class="chapter" data-level="" data-path="github-r.html"><a href="github-r.html#set-up-rstudio-github-connection"><i class="fa fa-check"></i>Set-up RStudio-Github connection</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="19" data-path="single-omic-analyses.html"><a href="single-omic-analyses.html"><i class="fa fa-check"></i><b>19</b> Single omic analyses</a></li>
<li class="chapter" data-level="20" data-path="data-transformations.html"><a href="data-transformations.html"><i class="fa fa-check"></i><b>20</b> Data transformations</a>
<ul>
<li class="chapter" data-level="20.1" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html"><i class="fa fa-check"></i><b>20.1</b> Transformations to account for statistical assumptions</a>
<ul>
<li class="chapter" data-level="" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#transforming-data-to-meet-normality-assumption"><i class="fa fa-check"></i>Transforming data to meet normality assumption</a></li>
<li class="chapter" data-level="20.1.1" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#data-transformation-compositional"><i class="fa fa-check"></i><b>20.1.1</b> Transformations to account for compositional data</a></li>
<li class="chapter" data-level="20.1.2" data-path="data-transformation-assumptions.html"><a href="data-transformation-assumptions.html#data-transformation-scaling"><i class="fa fa-check"></i><b>20.1.2</b> Transformations to account for scaling</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="21" data-path="unsupervised-exploration.html"><a href="unsupervised-exploration.html"><i class="fa fa-check"></i><b>21</b> Unsupervised exploration</a>
<ul>
<li class="chapter" data-level="21.1" data-path="cluster-analysis.html"><a href="cluster-analysis.html"><i class="fa fa-check"></i><b>21.1</b> Cluster analysis</a>
<ul>
<li class="chapter" data-level="21.1.1" data-path="cluster-analysis.html"><a href="cluster-analysis.html#hierarchical-clustering"><i class="fa fa-check"></i><b>21.1.1</b> Hierarchical clustering</a></li>
<li class="chapter" data-level="21.1.2" data-path="cluster-analysis.html"><a href="cluster-analysis.html#disjoint-clustering"><i class="fa fa-check"></i><b>21.1.2</b> Disjoint clustering</a></li>
</ul></li>
<li class="chapter" data-level="21.2" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html"><i class="fa fa-check"></i><b>21.2</b> Dimension reduction and ordination</a>
<ul>
<li class="chapter" data-level="21.2.1" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#pca"><i class="fa fa-check"></i><b>21.2.1</b> Principal Component Analysis (PCA)</a></li>
<li class="chapter" data-level="21.2.2" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#pcoa"><i class="fa fa-check"></i><b>21.2.2</b> Principal Coordinate Analysis (PCoA)</a></li>
<li class="chapter" data-level="21.2.3" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#nmds"><i class="fa fa-check"></i><b>21.2.3</b> Non-metric Multidimensional Scaling (NMDS)</a></li>
<li class="chapter" data-level="21.2.4" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#t-sne"><i class="fa fa-check"></i><b>21.2.4</b> t-Distributed Stochastic Neighbour Embedding (t-SNE)</a></li>
<li class="chapter" data-level="21.2.5" data-path="dimension-reduction-ordination.html"><a href="dimension-reduction-ordination.html#umap"><i class="fa fa-check"></i><b>21.2.5</b> Uniform manifold approximation and projection (UMAP)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="22" data-path="supervised-analysis.html"><a href="supervised-analysis.html"><i class="fa fa-check"></i><b>22</b> Supervised analysis</a>
<ul>
<li class="chapter" data-level="22.1" data-path="regression-methods.html"><a href="regression-methods.html"><i class="fa fa-check"></i><b>22.1</b> Regression methods</a>
<ul>
<li class="chapter" data-level="22.1.1" data-path="regression-methods.html"><a href="regression-methods.html#permanova"><i class="fa fa-check"></i><b>22.1.1</b> PERMANOVA</a></li>
<li class="chapter" data-level="22.1.2" data-path="regression-methods.html"><a href="regression-methods.html#anosim"><i class="fa fa-check"></i><b>22.1.2</b> ANOSIM</a></li>
<li class="chapter" data-level="22.1.3" data-path="regression-methods.html"><a href="regression-methods.html#redundancy-analysis"><i class="fa fa-check"></i><b>22.1.3</b> Redundancy analysis (RDA)</a></li>
<li class="chapter" data-level="22.1.4" data-path="regression-methods.html"><a href="regression-methods.html#canonical-correspondence-analysis"><i class="fa fa-check"></i><b>22.1.4</b> Canonical Correspondence Analysis (CCA)</a></li>
<li class="chapter" data-level="22.1.5" data-path="regression-methods.html"><a href="regression-methods.html#generalised-linear-modelling"><i class="fa fa-check"></i><b>22.1.5</b> Generalised linear modelling (GLM)</a></li>
<li class="chapter" data-level="22.1.6" data-path="regression-methods.html"><a href="regression-methods.html#generalised-linear-mixed-modelling"><i class="fa fa-check"></i><b>22.1.6</b> Generalised linear mixed modelling (GLMM)</a></li>
</ul></li>
<li class="chapter" data-level="22.2" data-path="classification-methods.html"><a href="classification-methods.html"><i class="fa fa-check"></i><b>22.2</b> Classification methods</a>
<ul>
<li class="chapter" data-level="22.2.1" data-path="classification-methods.html"><a href="classification-methods.html#random-forests"><i class="fa fa-check"></i><b>22.2.1</b> Random Forests (RF)</a></li>
<li class="chapter" data-level="22.2.2" data-path="classification-methods.html"><a href="classification-methods.html#support-vector-machines"><i class="fa fa-check"></i><b>22.2.2</b> Support Vector Machines (SVM)</a></li>
</ul></li>
</ul></li>
<li class="chapter" data-level="23" data-path="multi-omic-integration.html"><a href="multi-omic-integration.html"><i class="fa fa-check"></i><b>23</b> Multi-omic integration</a></li>
<li class="chapter" data-level="24" data-path="multi-staged-omics-integration.html"><a href="multi-staged-omics-integration.html"><i class="fa fa-check"></i><b>24</b> Multi-staged omics integration</a></li>
<li class="chapter" data-level="25" data-path="meta-dimensional-omics-integration.html"><a href="meta-dimensional-omics-integration.html"><i class="fa fa-check"></i><b>25</b> Meta-dimensional omics integration</a>
<ul>
<li class="chapter" data-level="25.1" data-path="multi-staged-omics-integration-concatenation-based.html"><a href="multi-staged-omics-integration-concatenation-based.html"><i class="fa fa-check"></i><b>25.1</b> Concatenation-based integration</a></li>
<li class="chapter" data-level="25.2" data-path="multi-staged-omics-integration-transformation-based.html"><a href="multi-staged-omics-integration-transformation-based.html"><i class="fa fa-check"></i><b>25.2</b> Transformation-based integration</a></li>
<li class="chapter" data-level="25.3" data-path="multi-staged-omics-integration-model-based.html"><a href="multi-staged-omics-integration-model-based.html"><i class="fa fa-check"></i><b>25.3</b> Model-based integration</a></li>
</ul></li>
<li class="part"><span><b>VI RESOURCES</b></span></li>
<li class="chapter" data-level="26" data-path="useful-links.html"><a href="useful-links.html"><i class="fa fa-check"></i><b>26</b> Useful links</a>
<ul>
<li class="chapter" data-level="" data-path="useful-links.html"><a href="useful-links.html#links-data"><i class="fa fa-check"></i>Data access</a></li>
<li class="chapter" data-level="" data-path="useful-links.html"><a href="useful-links.html#links-documentation"><i class="fa fa-check"></i>Documentation</a></li>
</ul></li>
<li class="chapter" data-level="27" data-path="references.html"><a href="references.html"><i class="fa fa-check"></i><b>27</b> References</a></li>
</ul>

      </nav>
    </div>

    <div class="book-body">
      <div class="body-inner">
        <div class="book-header" role="navigation">
          <h1>
            <i class="fa fa-circle-o-notch fa-spin"></i><a href="./">A Practical Guide to Holo-Omics</a>
          </h1>
        </div>

        <div class="page-wrapper" tabindex="-1" role="main">
          <div class="page-inner">

            <section class="normal" id="section-">
<div id="multi-staged-omics-integration-concatenation-based" class="section level2 hasAnchor" number="25.1">
<h2><span class="header-section-number">25.1</span> Concatenation-based integration<a href="multi-staged-omics-integration-concatenation-based.html#multi-staged-omics-integration-concatenation-based" class="anchor-section" aria-label="Anchor link to header"></a></h2>
<p>Concatenation-based integration combines multiple omic datasets, raw or pre-processed, into a single large matrix. One of the advantages of these approaches is their simplicity, since once the concatenation of multi-omic datasets is achieved, unsupervised and supervised analysis methods can be applied to the joint matrix, as in the case of the independent analysis of omic layers. Concatenation-based techniques offer a straightforward approach to utilising machine learning for the examination of both continuous and categorical data. Once the individual omics are concatenated, these methods can analyse all the combined features in an even-handed manner and pinpoint the most distinguishing features associated with a given phenotype. One of the main challenges of concatenation-based approaches is to ensure that the features of the different omic layers are comparable.</p>
<p>Several examples of unsupervised concatenation-based methods for multi-omic integration have been developed in recent years, most of them based on matrix-factorisation <span class="citation">[<a href="#ref-Reel2021-wb">58</a>]</span>. Joint non-negative matrix factorisation (Joint NMF) allowed integrating non-negative multi-omic data by decomposing the joint matrix into factors and loadings <span class="citation">[<a href="#ref-Zhang2021-sn">28</a>]</span>. Joint and Individual Variation Explained (JIVE) is an adaptation of NMF framework <span class="citation">[<a href="#ref-Lock2013-rq">59</a>]</span> which was later improved by Joint Bayes Factor (JBF) to handle the problems derived from the high sparsity of multi-omic datasets <span class="citation">[<a href="#ref-Ray2014-mn">60</a>]</span>. iCluster framework is based in similar principles to NMF but allows integration of datasets having negative values <span class="citation">[<a href="#ref-Shen2009-xr">61</a>]</span>. MoCluster <span class="citation">[<a href="#ref-Meng2016-hv">62</a>]</span>, RLAcluster <span class="citation">[<a href="#ref-Wu2015-qf">63</a>]</span> and iClusterBayes <span class="citation">[<a href="#ref-Mo2018-ty">64</a>]</span> have further developed the framework and improved it in terms of diversity of handled data types, computation speed and clustering accuracy. Multi-Omics Factor Analysis (MOFA) is another recent development that allows discovering the principal sources of variability across different omic datasets <span class="citation">[<a href="#ref-Argelaguet2018-dl">65</a>]</span>. Regarding supervised analyses, any of the algorithms for supervised analysis of single omic layers can be used to analyse concatenated multi omic data. RF <span class="citation">[<a href="#ref-Acharjee2016-uc">66</a>]</span>, SVM <span class="citation">[<a href="#ref-Li2017-qk">67</a>]</span>, LASSO regression <span class="citation">[<a href="#ref-Lee2017-bv">68</a>]</span> or DL <span class="citation">[<a href="#ref-Zhang2018-ty">69</a>]</span> algorithms have been used, among others, for concatenation-based supervised analysis in multi-omic literature.</p>
<div class="authorbox">
<p>Contents of this section were created by <a href="index.html#inaki-odriozola">Iñaki Odriozola</a> and <a href="index.html#antton-alberdi">Antton Alberdi</a>.</p>
</div>
</div>
<h3> References<a href="references.html#references" class="anchor-section" aria-label="Anchor link to header"></a></h3>
<div id="refs" class="references csl-bib-body">
<div id="ref-Zhang2021-sn" class="csl-entry">
28. Zhang B, Brock M, Arana C, Dende C, Oers NS van, Hooper LV, et al. Impact of <span>Bead-Beating</span> intensity on the genus- and <span>Species-Level</span> characterization of the gut microbiome using amplicon and complete <span>16S</span> <span class="nocase">rRNA</span> gene sequencing. Front Cell Infect Microbiol. 2021;11:678522.
</div>
<div id="ref-Reel2021-wb" class="csl-entry">
58. Reel PS, Reel S, Pearson E, Trucco E, Jefferson E. Using machine learning approaches for multi-omics data analysis: A review. Biotechnol Adv. 2021;49:107739.
</div>
<div id="ref-Lock2013-rq" class="csl-entry">
59. Lock EF, Hoadley KA, Marron JS, Nobel AB. <span>JOINT</span> <span>AND</span> <span>INDIVIDUAL</span> <span>VARIATION</span> <span>EXPLAINED</span> (<span>JIVE</span>) <span>FOR</span> <span>INTEGRATED</span> <span>ANALYSIS</span> <span>OF</span> <span>MULTIPLE</span> <span>DATA</span> <span>TYPES</span>. Ann Appl Stat. 2013;7:523–42.
</div>
<div id="ref-Ray2014-mn" class="csl-entry">
60. Ray P, Zheng L, Lucas J, Carin L. Bayesian joint analysis of heterogeneous genomics data. Bioinformatics. 2014;30:1370–6.
</div>
<div id="ref-Shen2009-xr" class="csl-entry">
61. Shen R, Olshen AB, Ladanyi M. Integrative clustering of multiple genomic data types using a joint latent variable model with application to breast and lung cancer subtype analysis. Bioinformatics. 2009;25:2906–12.
</div>
<div id="ref-Meng2016-hv" class="csl-entry">
62. Meng C, Helm D, Frejno M, Kuster B. moCluster: Identifying joint patterns across multiple omics data sets. J Proteome Res. 2016;15:755–65.
</div>
<div id="ref-Wu2015-qf" class="csl-entry">
63. Wu D, Wang D, Zhang MQ, Gu J. Fast dimension reduction and integrative clustering of multi-omics data using low-rank approximation: Application to cancer molecular classification. BMC Genomics. 2015;16:1022.
</div>
<div id="ref-Mo2018-ty" class="csl-entry">
64. Mo Q, Shen R, Guo C, Vannucci M, Chan KS, Hilsenbeck SG. A fully bayesian latent variable model for integrative clustering analysis of multi-type omics data. Biostatistics. 2018;19:71–86.
</div>
<div id="ref-Argelaguet2018-dl" class="csl-entry">
65. Argelaguet R, Velten B, Arnol D, Dietrich S, Zenz T, Marioni JC, et al. <span>Multi-Omics</span> factor analysis—a framework for unsupervised integration of multi-omics data sets. Mol Syst Biol. 2018;14:e8124.
</div>
<div id="ref-Acharjee2016-uc" class="csl-entry">
66. Acharjee A, Kloosterman B, Visser RGF, Maliepaard C. Integration of multi-omics data for prediction of phenotypic traits using random forest. BMC Bioinformatics. 2016;17 Suppl 5 Suppl 5:180.
</div>
<div id="ref-Li2017-qk" class="csl-entry">
67. Li S, Chen X, Liu X, Yu Y, Pan H, Haak R, et al. Complex integrated analysis of <span class="nocase">lncRNAs-miRNAs-mRNAs</span> in oral squamous cell carcinoma. Oral Oncol. 2017;73:1–9.
</div>
<div id="ref-Lee2017-bv" class="csl-entry">
68. Lee G, Bang L, Kim SY, Kim D, Sohn K-A. Identifying subtype-specific associations between gene expression and <span>DNA</span> methylation profiles in breast cancer. BMC Med Genomics. 2017;10 Suppl 1:28.
</div>
<div id="ref-Zhang2018-ty" class="csl-entry">
69. Zhang L, Lv C, Jin Y, Cheng G, Fu Y, Yuan D, et al. Deep <span>Learning-Based</span> <span>Multi-Omics</span> data integration reveals two prognostic subtypes in <span>High-Risk</span> neuroblastoma. Front Genet. 2018;9:477.
</div>
</div>
            </section>

          </div>
        </div>
      </div>
<a href="meta-dimensional-omics-integration.html" class="navigation navigation-prev " aria-label="Previous page"><i class="fa fa-angle-left"></i></a>
<a href="multi-staged-omics-integration-transformation-based.html" class="navigation navigation-next " aria-label="Next page"><i class="fa fa-angle-right"></i></a>
    </div>
  </div>
<script src="libs/gitbook-2.6.7/js/app.min.js"></script>
<script src="libs/gitbook-2.6.7/js/lunr.js"></script>
<script src="libs/gitbook-2.6.7/js/clipboard.min.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-search.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-sharing.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-fontsettings.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-bookdown.js"></script>
<script src="libs/gitbook-2.6.7/js/jquery.highlight.js"></script>
<script src="libs/gitbook-2.6.7/js/plugin-clipboard.js"></script>
<script>
gitbook.require(["gitbook"], function(gitbook) {
gitbook.start({
"sharing": {
"github": true,
"facebook": true,
"twitter": true,
"linkedin": false,
"weibo": false,
"instapaper": false,
"vk": false,
"whatsapp": false,
"all": ["facebook", "twitter", "linkedin", "weibo", "instapaper"]
},
"fontsettings": {
"theme": "white",
"family": "sans",
"size": 2
},
"edit": null,
"history": {
"link": null,
"text": null
},
"view": {
"link": null,
"text": null
},
"download": null,
"search": {
"engine": "lunr",
"options": null
},
"toc": {
"collapse": "section",
"scroll_highlight": true
},
"toolbar": {
"position": "fixed"
},
"info": true
});
});
</script>

</body>

</html>
